src/core/qfieldcloud/qfieldcloudproject.cpp

```diff
@@ -498,7 +499,7 @@ void QFieldCloudProject::downloadAttachment( const QString &fileName )
 {
   if ( !mAttachmentsFileTransfers.contains( fileName ) )
   {
-    mAttachmentsFileTransfers.insert( fileName, FileTransfer( fileName, 0, mId ) );
+    mAttachmentsFileTransfers.insert( fileName, FileTransfer( fileName, 0, mId, FileUtils::fileEtag( fileName ) ) );
```

Comment:
Why don't we use `prepareDownloadTransfer` here?

Reply:
@suricactus , it didn't need the extra logic from that function to begin with.